client/js/widgets/widget.js

```diff
@@ -723,6 +727,9 @@ export class Widget extends StateManaged {
           c.filter(w=>w.p('type')=='pile').forEach(w=>c.push(...w.children()));
           c = c.filter(w=>w.p('type')!='pile');
           collections[a.collection] = [...new Set(c)];
+
+          if (a.key)
```

Comment:
Feels weird that it's just called `key` because it's not in the context of sorting here.